test/configCases/module-name/different-issuers-for-same-module/loader-a.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(src) {
+	return `module.exports = "loader-a"`;
```

Comment:
You can get access to the `module.id` in a module:

``` js
return `module.exports = "loader-a " + module.id`;
```

That makes it easier to test it.